### PR TITLE
feat: Adaptation log specification rectification

### PR DESCRIPTION
**Pull request by starhcq**
Adaptation log specification rectification

Log: Adaptation log specification rectification
Task: https://pms.uniontech.com/task-view-292081.html

### CODE REVIEW

File: application/main.cpp [hunk 1]
```diff
@@ -26,6 +26,8 @@
 DWIDGET_USE_NAMESPACE
 DCORE_USE_NAMESPACE
 
+Q_LOGGING_CATEGORY(dl,"deepin.log.viewer")
```
Comment (rb-union) [inline application/main.cpp:29]: 好像没有用

Comment (starhcq) [reply application/main.cpp]: 已移除

**Review by hundundadi — COMMENTED** [review]
[是否满足兼容性要求] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

File: application/main.cpp [hunk 2]
```diff
@@ -103,7 +111,6 @@ int main(int argc, char *argv[])
         auto systemEnv = QProcessEnvironment::systemEnvironment();
         QString XDG_SESSION_TYPE = systemEnv.value(QStringLiteral("XDG_SESSION_TYPE"));
         QString WAYLAND_DISPLAY = systemEnv.value(QStringLiteral("WAYLAND_DISPLAY"));
-        qDebug() << "XDG_SESSION_TYPE:" << XDG_SESSION_TYPE << "---WAYLAND_DISPLAY:" << WAYLAND_DISPLAY;
```
Comment (rb-union) [inline application/main.cpp:106]: cppehck的问题，变量多了。

**Review by rb-union — APPROVED** [review]
/+1

**Pull request merged**